Project_1/Styles.css

```
@@ -51,14 +50,21 @@ p {
     line-height: 2.2;
 }
 
-/*i am not sure how to edit either image individually, so right now the Author photo is very much warped /*
+p #pullquote {
```

If you make the changes I suggest in the HTML, remember to update this selector, too. You'll need to change it like so:
```suggestion
#pullquote p {
```